haystack/components/generators/openai_utils.py

```diff
@@ -1,23 +0,0 @@
-# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
-#
-# SPDX-License-Identifier: Apache-2.0
-
-from typing import Dict
-
-from haystack.dataclasses import ChatMessage
-
-
-def _convert_message_to_openai_format(message: ChatMessage) -> Dict[str, str]:
```

Comment:
this was practically based on the legacy `ChatMessage` - I'm introducing a better method in the `ChatMessage dataclass` (in preparation for #8638)